src/app/domain/entities/manga/manga-entity.ts

```diff
@@ -1,6 +1,23 @@
 import { DomainEntity } from '../base/domain-entity';
 
+interface MangaProps {
+  name: string;
+  total_pages: number;
+}
+
 export class MangaEntity extends DomainEntity {
   name!: string;
-  pages!: number;
+  total_pages!: number;
+  actual_page?: number;
```

Comment:
Why `actual_page` isn't in `MangaProps` interface?

Reply:
I don't like having to declare these attributes here and create an interface with the same attributes to pass it on constructor. Maybe there is a better way to do this.

Reply:
> Why actual_page isn't in MangaProps interface?
Because not is a attribute important to creating a new mangá. I believe that is unnecessary.